src/components/sokperson/SokPerson.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import {
+  BodyShort,
+  Box,
+  Button,
+  Heading,
+  HStack,
+  Modal,
+  TextField,
+  VStack,
+} from '@navikt/ds-react';
+import { useSokPerson } from '@/data/personoversiktHooks';
+import { PersonOversiktStatusDTO } from '@/api/types/personoversiktTypes';
+import { SokDTO } from '@/api/types/sokDTO';
+import SokPersonResultat from '@/components/sokperson/SokPersonResultat';
+import { MagnifyingGlassIcon } from '@navikt/aksel-icons';
+import { isNumeric, removePunctuation } from '@/utils/stringUtil';
+import { parseDateString } from '@/utils/dateUtils';
+
+const texts = {
+  buttonText: 'Søk etter sykmeldt',
+  header: 'Søk etter sykmeldt',
+  info:
+    'Her kan du søke opp sykmeldte personer basert på initialer og fødselsdato.',
+  validation: {
+    initials: 'Vennligst angi gyldige initialer',
+    birthdate: 'Vennligst angi en gyldig fødselsdato',
+  },
+};
+
+export default function SokPerson() {
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
+  const [nameInitials, setNameInitials] = useState<string>('');
+  const [birthdate, setBirthdate] = useState<string>('');
+  const sokPerson = useSokPerson();
```

Comment:
Går an å gjøre `const { mutate, data, isLoading, isError } = useSokPerson();` her så kan man bruke `data` i stedet for å ha en egen state for `searchResult`. Og så bør vi vise en feilmelding ved `isError`.